package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "3.138.16",
+  "version": "3.139.16",
```

Comment:
How about `3.139.0` or `3.138.17`? Either is fine with me.
We need to better establish a better convention on how to bump versions (that's mostly a note to myself).